solr/contrib/ltr/src/java/org/apache/solr/ltr/store/rest/ManagedModelStore.java

```diff
@@ -110,7 +115,7 @@ public void loadStoredModels() {
       final List<Map<String,Object>> up = (List<Map<String,Object>>) managedData;
```

Comment:
can we have better naming for `u` and `up` ? something like: 
`for (final Map<String,Object> modelParams : modelsParams)` ?